.gitignore

```diff
@@ -151,6 +151,8 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
+.DS_Store
```

Comment:
You probably do not need `.gitignore `

Reply:
I can take it out if you'd like. If you make the docs on Mac, it'll make one of these files:
```
$ find . -name ".DS_Store"
./docs/.DS_Store
```
so we'd just have to make sure people manually remove them or dont add them to git